src/Capgemini.PowerApps.PackageDeployerTemplate/Config/ConfigDataStorage.cs

```diff
@@ -76,6 +76,14 @@ public ConfigDataStorage()
         [XmlArray("dataimports")]
         [XmlArrayItem("dataimport")]
         public DataImportConfig[] DataImports { get; set; }
+             
+        [XmlArray("connectionreferences")]
+        [XmlArrayItem("connectionreference")]
+        public ConnectionReference[] ConnctionReferences { get; set; }
+
+        [XmlArray("flowstodeactivate")]
+        [XmlArrayItem("flowtodeactivate")]
+        public string[] Flows { get; set; }
```

Comment:
This name could be more explicit like `FlowsToDeactivate` to avoid confusion as I did in `PackageTemplateBase`.

Reply:
Yes agree originally was going to be used for something else as well as activation.